gtp2ogs.js

```diff
@@ -1619,20 +1673,21 @@ class Connection {
             conn_log("Unhandled rules: " + notification.rules + ", rejecting challenge");
             return { reject: true, msg: "The " + notification.rules + " rules are not allowed for this bot, please choose allowed rules such as chinese rules. " };
         }
-        
-        
-        function timespanToDisplayString(timespan) {
-            let ss = timespan % 60;
-            let mm = Math.floor(timespan / 60 % 60);
-            let hh = Math.floor(timespan / (60*60) % 24);
-            let dd = Math.floor(timespan / (60*60*24));
-            let text = ["days", "hours", "minutes", "seconds"];
-            return [dd, hh, mm, ss]
-                .map((e, i) => e === 0 ? "" : `${e} ${text[i]}`)
-                .filter(e => e !== "")
-                .join(" ");
+
+        // ** first we eliminate absolute and simple time control : they don't support the use of a period time
+        // using absolute or simple (no period time) is likely to make bots timeout or play way too fast 
+        /* "fischer" , "simple", "byoyomi" , "canadian" , "absolute"*/
+        let TimeControlString = String(t.time_control);
+        if (TimeControlString === "absolute") {
+            conn_log("Minimum and Maximum main time is not supported in time control " + TimeControlString);
+            return { reject: true, msg: "Period time management is not supported in the time control " + TimeControlString + " , please choose a time control that supports the use of a period time, such as byoyomi,fischer,canadian." };
```

Comment:
I have some doubts about outright rejecting `absolute`, but that could be something we fix in another commit.

NOTE: This only reject it if we're using min and max time here right? I cannot see that clearly in the PR in this github view.

Reply:
no, in the past absolute time was always rejected independently from anything (same as why all rules default to chinese on ogs)

what i added here is that i rejected simple time too due to its confusing behaviour : 

it has a maintime with `t.main_time`
but it also has a period time with `t.per_move`

so i dont know how it should be handled with the arguments, hence why i removed it until we clear that out